easy_phi/tests/handlers_test.py

```diff
@@ -97,12 +101,77 @@ def test_list_scpi_commands(self):
         self.failIf(response.error)
         response_obj = json.loads(response.body)
 
-        self.assertTrue(isinstance(response_obj, list),
-                        "module_scpi_list expected to return list of commands")
+        self.assertIsInstance(
+            response_obj, list,
+            "module_scpi_list expected to return list of commands")
```

Comment:
The first command gives me:
<code>
sudo ln -s scripts/modules_conf_patches /etc/easy_phi/modules_conf_patches.conf
[sudo] password for parallels: 
ln: failed to create symbolic link ‘/etc/easy_phi/modules_conf_patches.conf’: No such file or directory
</code>

The second command just runs the app with the params but not tests, so the test is still failing.